src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -87,7 +89,11 @@ public virtual async Task ProcessLogQueue(object state)
             while (!_cancellationTokenSource.IsCancellationRequested)
             {
                 await InternalProcessLogQueue();
-                await Task.Delay(TimeSpan.FromSeconds(FlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                await Task.Delay(TimeSpan.FromSeconds(_currentFlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                if (_currentFlushFrequencySeconds < MaxFlushFrequencySeconds)
+                {
+                    _currentFlushFrequencySeconds = Math.Min(MaxFlushFrequencySeconds, _currentFlushFrequencySeconds * 2);
```

Comment:
why do we multiplying 2 here?

Reply:
This is the backoff mechanism for fast file logging. So the delay on the Task is set to the following sequence: 1, 2, 4, 8, 16, 30 and maxes out at 30